var/spack/repos/builtin/packages/cairo/package.py

```diff
@@ -28,9 +28,10 @@
 class Cairo(AutotoolsPackage):
     """Cairo is a 2D graphics library with support for multiple output
     devices."""
-    homepage = "http://cairographics.org"
-    url      = "http://cairographics.org/releases/cairo-1.14.0.tar.xz"
+    homepage = "https://cairographics.org"
```

Comment:
Is the change to https necessary?  The package is checksummed, and many systems have bad SSL configurations that will fail on fetching this.